model/SearchEngine/Driver/Elasticsearch/ElasticSearchConfig.php

```diff
@@ -47,48 +55,69 @@ public function __construct(ServiceOptionsInterface $serviceOptions)
     public function getHosts(): array
     {
         $hosts = [];
-        foreach ($this->serviceOptions->get(self::class, self::OPTION_HOSTS) as $host) {
-            if (is_array($host) && isset($host['host'])) {
-                $hosts[] = ($host['scheme'] ?? 'http') . '://' . $host['host'] . ':' . ($host['port'] ?? 9200);
-            } else {
-                $hosts[] = $host;
+        if ($this->serviceOptions->get(self::class, self::OPTION_HOSTS)) {
+            foreach ($this->serviceOptions->get(self::class, self::OPTION_HOSTS) as $host) {
+                if (is_array($host) && isset($host['host'])) {
+                    $hosts[] = ($host['scheme'] ?? 'http') . '://' . $host['host'] . ':' . ($host['port'] ?? 9200);
+                } else {
+                    $hosts[] = $host;
+                }
             }
+            return $hosts;
+        }
+
+        if (getenv(self::ENV_OPTION_HOSTS)) {
```

Comment:
@tikhanovichA Dont you think that env vars should be priority and fallback to config? Imagine if Ops set the env vars later, after we execute the migrations. Then we would have a problem